frontend/src/tooltip.ts

```diff
@@ -0,0 +1,33 @@
+import type { TippyOptions } from "vue-tippy";
+import { type Instance } from "tippy.js";
+import { makeLabel } from "@/util/string";
+
+export const tippyOptions: {
```

Comment:
Moved this from main.ts to here to separate it better. Also added a few options.